grist-doc-wk-exporter/README.md

```diff
@@ -0,0 +1,17 @@
+# Prometheus exporter for Grist doc workers
+
+**Status: ✅ Ready to be used in production**
+
+This simple service allows exporting data from doc workers as a prometheus metrics endpoint.
+It is meant to run as a sidecar container.
+
+In its first version, it is used to track pod cgroup memory max and usage, and doc memory and cpu usage.
+
+## Setup
+
+Using `mise`:
+
+- [install `mise`](https://mise.jdx.dev/getting-started.html)
+- run `mise i`
+- run `uv sync`
+- run `uv run main.py`
```

Comment:
In what case would it be useful to run it locally, outside of kubernetes? If there is no such case, would this setup be useful?

What do you think of rather suggesting a way to setup this sidecar in kubernetes?

Reply:
It's useful when developping